app/components/curriculum-inventory/publication-status.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+
+export default class CuriculumInventoryPublicationStatusComponent extends Component {
+  get textKey(){
+    if (this.args.item.isScheduled) {
```

Comment:
inventory reports really only have one status, that's `Finalized`.

should therefore be something like: 

```javascript
if (this.args.item.isScheduled) {
  return 'general.finalized';
}  else {
  return 'general.draft';
}
```